### DISCUSSION TITLE
GitHub Education not accepting my documents.

### DISCUSSION THREAD

**Discussion by Walter-Hartwell-White** [community/community#58044] [GitHub Education]
### Hi everyone,

Hello @[GitHubEducation](https://github.com/community) I have uploaded my dated school id but GitHub is not accepting it. Probably something to do with the automated test. I have tried it with various ways, example, I even tried holding the ID with my webcam on, it still rejected it.
How else can I submit the document, is there a manual verification?

Thanks for the responses.

**Comment by samus-aran** [comment]
Hi @jinu2984  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
